src/formatter.js

```diff
@@ -126,6 +142,12 @@ class PrettyFormatter extends Formatter {
 
     drawStep(step) {
         let line = this.indent + chalk.bold(this.keywords[step.type] ?? '*') + ' ' + step.stepText;
+        line += ' ' + chalk.gray(step.location);
+        if (this.showLogs) {
+            for (const log of step.logs) {
+                line += '\n' + this.indent + log.body;
+            }
+        }
         if (step.argument && step.argument.dataTable) {
             line += '\n' + this.drawDataTable(step.argument.dataTable)
```

Comment:
it is similar to comment above with ${}

Reply:
all concats via + will be refactored later